pkg/crd/cloudprivateipconfig/v1/types.go

```diff
@@ -0,0 +1,61 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +genclient:noStatus
+// +resource:path=cloudprivateipconfig
+// +kubebuilder:resource:shortName=cpip,scope=Cluster
```

Comment:
Don't use a short-name; this isn't a user-facing type.

Reply:
I meant to have it like this a short-cut for us when debugging this stuff. What does it hurt if it's specified?

Reply:
shortnames are a limited space. In the past, we've had pushback when adding them, unless they are a core type that a user would type themselves.

For debugging, just set a shell alias.